cpp/include/cudf/filling.hpp

```diff
@@ -103,26 +103,21 @@ std::unique_ptr<column> fill(
  * ```
  * @p count should not have null values; should not contain negative values;
  * and the sum of count elements should not overflow the size_type's limit.
- * It is undefined behavior if @p count has negative values or the sum overflows
- * and @p check_count is set to false.
+ * It is undefined behavior if @p count has negative values or the sum overflows.
```

Comment:
(No antecedent for "It")

```suggestion
 * The behavior of this function is undefined if @p count has negative values or the sum overflows.
```